CONTRIBUTING.md

```diff
@@ -17,41 +16,59 @@ This guide will help you get started and outline some things you should know whe
   (install [directly](https://www.ruby-lang.org/en/documentation/installation),
   or via a ruby manager like [rvm](https://rvm.io/)
   or [rbenv](https://github.com/sstephenson/rbenv))
-  * Language the tests are written in
+  * language the tests are written in
 * [ShellCheck](https://github.com/koalaman/shellcheck)
-  * Used in the linting process to find common errors in the Bash code
+  * used in the linting process to find common errors in the Bash code
 
 
 ## Setup
 
+* install the [requirements](#requirements)
 * fork and clone the repository to your machine
-* `bundle` to install ruby gems
+* run `bundle` to install ruby gems
+* run `bundle exec rake` to make sure all tests pass
 
 
 ## Testing
 
-* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
-* All features need to have comprehensive test coverage
-* Source code and test files must pass the linters
+* tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* all features need to have comprehensive test coverage
+* source code and test files must pass the linters
 
 ```bash
-# rake tasks
-rake         # Run linters and feature tests
-rake format  # Run formatters (fixes some lint errors)
-rake lint    # Run linters
-rake test    # Run feature tests
+# run all tests
+bundle exec rake
```

Comment:
Oh. I personally believe gemsets are better than `bundle exec`. Yes it creates redundancy but its far easier to deal with. Gemsets are also similar to how npm works out of the box where dependencies are installed on a per project basis